integrations/oci/sdk/runtime/src/test/java/io/helidon/integrations/oci/sdk/runtime/OciAuthenticationDetailsProviderTest.java

```diff
@@ -117,23 +117,24 @@ void testToNamedProfile() {
     @Test
     void authStrategiesAvailability() {
         Config config = OciConfigTest.createTestConfig(
-                        OciConfigTest.ociAuthConfigStrategies(OciAuthenticationDetailsProvider.TAG_AUTO),
+                        OciConfigTest.ociAuthConfigStrategies(OciAuthenticationDetailsProvider.VAL_AUTO),
                         OciConfigTest.ociAuthSimpleConfig("tenant", "user", "phrase", "fp", null, null, "region"))
                 .get(OciConfig.CONFIG_KEY);
         OciConfig cfg = OciConfig.create(config);
         assertThat(OciAuthenticationDetailsProvider.AuthStrategy.AUTO.isAvailable(cfg),
                    is(true));
         assertThat(OciAuthenticationDetailsProvider.AuthStrategy.CONFIG.isAvailable(cfg),
                    is(false));
-        assertThat(OciAuthenticationDetailsProvider.AuthStrategy.CONFIG_FILE.isAvailable(cfg),
-                   is(false));
+        // this code is dependent upon whether and OCI config-file is present - so leaving this commented out intentionally
```

Comment:
You can use JUnit `Assumptions` instead

Reply:
I'd prefer not to check for the presence of the ~/.oci/config file, since that is breaking encapsulation into the OCI implementation.